src/pages/index.js

```diff
@@ -108,30 +106,23 @@ document.addEventListener("DOMContentLoaded", () => {
       .updateUserInfo(header, description)
       .then((res) => {
         userInformation.setUserInfo(res);
-        userInformation.setUserAvatar(res);
+        profileEditForm.renderLoading(false);
```

Comment:
`        profileEditForm.renderLoading(false);
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`. You don't need to duplicate the same  code in `catch` and `then`